### PR TITLE
Clarify how data and cache updates works

### PR DESCRIPTION
**Pull request by PaulWessel**
We never spell that out anywhere but here we do.

### CODE REVIEW

File: doc/rst/source/datasets/earth-daynight.rst [hunk 1]
```diff
@@ -66,6 +66,16 @@ to the geotiff files on the remote server. **Note**: This data set is experiment
 format and delivery is likely to change in the future (e.g., via image tiles).  To make the
 files as small as possible we have also downgraded them from 24-bit to 8-bit indexed images.
 
+Data Updates
```
Comment (seisman) [inline doc/rst/source/datasets/earth-daynight.rst:69]: It's better to write it in one file (e.g., `dataset_updates.rst_`) and include it in all other files.

Comment (PaulWessel) [reply doc/rst/source/datasets/earth-daynight.rst:69]: you are wiser than me. Will do.

File: doc/rst/source/datasets/earth-daynight.rst [hunk 2]
```diff
@@ -66,6 +66,8 @@ to the geotiff files on the remote server. **Note**: This data set is experiment
 format and delivery is likely to change in the future (e.g., via image tiles).  To make the
 files as small as possible we have also downgraded them from 24-bit to 8-bit indexed images.
 
+.. include:: data-updating.rst_
```
Comment (seisman) [inline doc/rst/source/datasets/earth-daynight.rst:69]:
```suggestion
.. include:: ../data-updating.rst_
```

File: doc/rst/source/datasets/earth-masks.rst [hunk 3]
```diff
@@ -70,3 +70,5 @@ To make a pixel-registered mask that has NaNs on land and 1 in the oceans, use::
 Finally, to make a pixel-registered mask that is 1 for lakes and NaN elsewhere, try::
 
     gmt grdmath @earth_mask_01m 2 EQ 0 NAN = lakes_and_NaN.grd
+
+.. include:: data-updating.rst_
```
Comment (seisman) [inline doc/rst/source/datasets/earth-masks.rst:74]:
```suggestion
.. include:: ../data-updating.rst_
```

File: doc/rst/source/datasets/earth-relief.rst [hunk 4]
```diff
@@ -126,6 +126,8 @@ To get all the data for Earth but only for 30 arc sec and coarser, and leave as
 
 As shown in the table, the largest dataset may take some time to download the data from GMT server. Be patient!
 
+.. include:: data-updating.rst_
```
Comment (seisman) [inline doc/rst/source/datasets/earth-relief.rst:129]:
```suggestion
.. include:: ../data-updating.rst_
```

**Pull request merged**